src/TMTProductizer.UnitTests/Mocks/MockUtils.cs

```diff
@@ -0,0 +1,28 @@
+
+namespace TMTProductizer.UnitTests.Mocks;
+
+public static class MockUtils
+{
+
+    public static string GetTMTTestResponse()
+    {
+        try
+        {
+            // Open the text file using a stream reader.
+            using (var sr = new StreamReader("../../src/TMTProductizer.UnitTests/Mocks/testTMTResponse.json"))
```

Comment:
Tämän JSON tiedoston lataaminen ei jostain syystä toimi omalla koneella eli testit feilaa. Parempi tapa olisi käyttää Moq tai builder patternia tuons responsen mockittamiseen, mutta en ehkä lähtisi sitä tällä tiketillä enään tekemään, vaan testejä voi parantaa myöhemmin.

Reply:
Näin on, tuon viitteen pitäisi nyt olla kuosittu kuitenkin.